GiEnJul/Utilities/EmailTemplates/EmailTemplateBuilder.cs

```diff
@@ -26,13 +26,34 @@ public async Task<EmailTemplate> GetEmailTemplate(EmailTemplateName name, Dictio
 
             var resourceManager = new ResourceManager(typeof(EmailTitles));
             var templatePath = string.Format("{0}Utilities{0}EmailTemplates{0}Files{0}", Path.DirectorySeparatorChar);
-            var headerImage = await File.ReadAllTextAsync($"{AppContext.BaseDirectory}{templatePath}familyTop.0ff49710.svg");
             var content = await File.ReadAllTextAsync($"{AppContext.BaseDirectory}{templatePath}{name}.html");
-            var style = await File.ReadAllTextAsync($"{AppContext.BaseDirectory}{templatePath}TemplateStyle.css");
-            style = "<style>" + style + "</style>";
-            content = "<!DOCTYPE html> <html>" + style + content + "</html>";
 
-            data.Add("headerImage", headerImage);
+            // Read style, apply to body
+            string[] emailCSS = { // Read from css file instead
+                "background-color: rgb(224, 243, 244);",
+                "color: rgba(0, 0, 0, 0.54);",
+                "max-width: 70em;",
+                "padding: 15px;",
+                "margin: auto;",
+
+            };
+            var emailStyle = "";
+            for (int i = 0; i < emailCSS.Length; i++)
+            {
+                emailStyle += emailCSS[i];
+            }
+            emailStyle = "\"" + emailStyle + "\"";
```

Comment:
string.Join can be used to combine a list/array of strings into a single one
```suggestion
            var emailStyle = $"\"{string.Join("", emailCSS)}\"";
```